feed-integration-webapp/src/main/resources/crater.properties

```diff
@@ -0,0 +1,11 @@
+con.content_type = application/json
+con.accept  = application/json
+con.setDoOutput = true
+con.setDoInput = true
+
+crater.currencyid = 13 #for INR
+crater.url = https://payments-dev.lite.mybahmni.in
+crater.company = 1
+crater.device_name = mobile_app
+crater.username = superman@bahmni.org
```

Comment:
Use a bridge specific credential, rather than using a generic superadmin credential.

Reply:
Similar to above issue, I have asked Venu for the credentials.